react/demo/src/daisyui/samples/alert/Default.route.tsx

```diff
@@ -0,0 +1,38 @@
+import type {AlertApi} from '@agnos-ui/react-headless/components/alert';
+import {useRef} from 'react';
+import {Alert} from './Alert';
+
+const DefaultAlertDemoDaisyUI = () => {
+	const refAlert = useRef<AlertApi>(null);
```

Comment:
React is the only example that exports the api (through a forwardRef) and uses it to reopen the first alert...
I would prefer the three examples to be consistent with each other.
Pagination and Rating also do not use forwardRef right now, so Alert would be the only one...

To be consistent between components / frameworks and keep it as simple as possible, I would propose to remove the api export in the react example (so no more forwardRef).
In this sample, it would become:
```tsx
const DefaultAlertDemoDaisyUI = () => {
  const [visible, setVisible] = useState(true);
  return (
    <>
      <button className="btn btn-primary" onClick={() => setVisible(true)}>
        Reset Alert
      </button>
      <div className="flex flex-col gap-3 mt-3">
        <Alert className="alert-success" dismissible={true} visible={visible} onVisibleChange={setVisible}>
          <svg></svg>
          <span>Alert is a success!</span>
        </Alert>
        // second alert
      </div>
    </>
  );
};
```